net/wfb-ng/Makefile

```diff
@@ -0,0 +1,45 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wfb-ng
+PKG_VERSION:=24.9.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng.git
+PKG_SOURCE_VERSION:=wfb-ng-24.09-openwrt-preview
+PKG_MIRROR_HASH:=bb29c4b8b5f8eaefb933060b180c8ff58bc6c9d216bc97d4dad3c10c8c0c5e69
```

Comment:
You can not use Git sources, because your tags contains prefix "release".

There could be workaround, etc. If you would tag it normally, then you could use:

```suggestion
PKG_NAME:=wfb-ng
PKG_VERSION:=SemVer
PKG_RELEASE:=1

PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng.git
PKG_SOURCE_VERSION:=($PKG_VERSION)
PKG_MIRROR_HASH:=XXXX
```

So, the version will be equal to the tag.

Also, some packages are using this:

```suggestion
PKG_NAME:=wfb-ng
PKG_RELEASE:=1

PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng.git
PKG_SOURCE_DATE:=latestDateofGitCommit
PKG_SOURCE_VERSION:=latestGitCommit
PKG_MIRROR_HASH:=bb29c4b8b5f8eaefb933060b180c8ff58bc6c9d216bc97d4dad3c10c8c0c5e69
```
Basically, this removes your version, because your versioning does not match the commit/branch, etc. Versioning of the package will contain the PKG_SOURCE_VERSION hash in short form.

What is preferred in your Makefile is this as I said it in the beginning:
```suggestion
PKG_NAME:=wfb-ng
PKG_VERSION:=24.08.1
PKG_RELEASE:=1

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng/releases/download/$(PKG_NAME)-$(PKG_VERSION)/
PKG_HASH:=sha256sum
```

Not tested, everything was followed by the available documentation and everything can be found there since I still does not find usage for this package and I dont think that we need to merge this package.

Reply:
https://github.com/svpcom/wfb-ng/wiki/Distributed-operation
this howto refers to `wfb-ng` openwrt package

Reply:
@BKPepe I've switched to using tgz from release instead of git. Now it should meet the requirements.